docs/nddata/utils.rst

```diff
@@ -0,0 +1,257 @@
+.. _nddata_utils:
+
+astropy.nddata.utils
+====================
+
+Overview
+--------
+
+The `astropy.nddata.utils` module includes general utility functions
+for array operations.
+
+
+Cutout Images
+-------------
+
+Getting Started
+^^^^^^^^^^^^^^^
+
+The `~astropy.nddata.utils.Cutout` class can be used to create a
+postage stamp cutout image from a 2D array.  If an optional
+`~astropy.wcs.WCS` object is input to `~astropy.nddata.utils.Cutout`,
+then the `~astropy.nddata.utils.Cutout` object will contain an updated
+`~astropy.wcs.WCS` corresponding to the cutout array.
+
+First, let's create a simple 2D data array::
+
+    >>> import numpy as np
+    >>> from astropy.modeling.models import Gaussian2D
+    >>> y, x = np.mgrid[0:500, 0:500]
+    >>> data = Gaussian2D(1, 50, 100, 10, 5, theta=0.5)(x, y)
+
+Next, let's display the image:
+
+.. doctest-skip::
+
+    >>> import matplotlib.pyplot as plt
+    >>> plt.imshow(data, origin='lower')
+
+.. plot::
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+    from astropy.modeling.models import Gaussian2D
+    y, x = np.mgrid[0:500, 0:500]
+    data = Gaussian2D(1, 50, 100, 10, 5, theta=0.5)(x, y)
+    plt.imshow(data, origin='lower')
+
+Now let's create a cutout array for the single object in this image.
+We create a cutout array centered at position ``(100.1, 49.7)`` (``(y,
```

Comment:
I find this hard to read:
![image](https://cloud.githubusercontent.com/assets/852409/8621876/59a4fdd2-2728-11e5-91fc-05363345946f.png)

Maybe instead write it with an equals sign?
```
(y, x) = (100.1, 49.7)
```